pipelines/optimus/example_test_inputs.json

```diff
@@ -1,19 +1,19 @@
 {
   "Optimus.fastq_inputs": [
     [
-      "gs://broad-dsde-mint-dev-teststorage/10x/demo/fastqs/pbmc8k_S1_L007_R1_001.fastq.gz",
-      "gs://broad-dsde-mint-dev-teststorage/10x/demo/fastqs/pbmc8k_S1_L007_R2_001.fastq.gz",
-      "gs://broad-dsde-mint-dev-teststorage/10x/demo/fastqs/pbmc8k_S1_L007_I1_001.fastq.gz"
+      "gs://hca-dcp-mint-test-data/10x/demo/fastqs/pbmc8k_S1_L007_R1_001.fastq.gz",
```

Comment:
The inputs for optimus have changed so there are now three different arrays for fastq files instead of an array of arrays: https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/optimus/Optimus.wdl#L16

I just noticed this after looking at Marcus's PR that changes the test inputs file: https://github.com/HumanCellAtlas/skylab/pull/86/files#diff-c53d14ec9a922a0786f2647a9480b7b3R4

Reply:
Confirmed, could you please fix my screw-up and update this to be: 

```
  "Optimus.r1": [
      "gs://hca-dcp-mint-test-data/10x/demo/fastqs/pbmc8k_S1_L007_R1_001.fastq.gz",
      "gs://hca-dcp-mint-test-data/10x/demo/fastqs/pbmc8k_S1_L007_R1_001.fastq.gz"
  ],
  "Optimus.r2": [
      "gs://hca-dcp-mint-test-data/10x/demo/fastqs/pbmc8k_S1_L007_R2_001.fastq.gz",
      "gs://hca-dcp-mint-test-data/10x/demo/fastqs/pbmc8k_S1_L007_R2_001.fastq.gz"
  ],
  "Optimus.i1": [
      "gs://hca-dcp-mint-test-data/10x/demo/fastqs/pbmc8k_S1_L007_I1_001.fastq.gz",
      "gs://hca-dcp-mint-test-data/10x/demo/fastqs/pbmc8k_S1_L007_I1_001.fastq.gz"
  ]

```

Reply:
@ambrosejcarr should this be data from the 10x or Optimus folder here?

Reply:
10x is the data type, so is appropriate here.

Reply:
Okay, I fixed that, could use another look @ambrosejcarr.